Source/Scene/PointCloud.js

```diff
@@ -1,4 +1,3 @@
-import arraySlice from "../Core/arraySlice.js";
 import BoundingSphere from "../Core/BoundingSphere.js";
```

Comment:
I'm still trying to figure out why the `TimeDynamicPointCloud` unit tests are failing. Taking down some notes here. 

I checked the camera setup and frustum, nothing noticeably different between the unit test and [this sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=rVVbb9s2FP4rhF8qFTJFyte4TrDA2dAOHRI0xh4WBQMjMTERijQoym065L/vkLpm84ICCyBIJD+e79yPNrwUVYF//rbnRhRcWSZ/4cxWhpeYK3Yn+W8653KIo1N0z2TJP6QqVQdm0EHwr9zAseJf0aYm/N2fBeko8/uNBkmhuElHEforVQiVO13J/FyJglm+QtZUPErVc9ix5nBeAumNu52OEkKXY7IY05Mtna3ockXIH0D2nyCevQbTV8HXZZNXwVb2tnOkMmLgB8YxPNes2Et+wSyL64BNLrZC8jK+0kLZjdRVPlhuIfIXT4oVIosJ3itbdhb8Xzr6tnTJ29JN3pZu2tPV6bmvVGaFVq7Y2IZJeceyxwBEuTkwGSGhcv4trAvWcOgKVa+RS+rKZ/bG37n1Fj4D6XObdguKPzVMGy0lrzWdth2yPYrje6OLT6VezgkFF/m5MewpcOSNYtFj5apukqhGhj6sXuxq41LVN9e+i8/Lth1Eqw9h4FW3UQGtx13zWjKpnfZ6JGC/8+elfZLQ5gNVw9xdOzRoPPS2XYvvcJ2SxrdMS21W6J3/wlQxPE9H4bvasbAdHI3WMuOK4z3MK2HFAQYZy/Og97iPQmmZsX1Gfq2kYMpFdpiFwMf4htx6DV5K739QyL+x5OrB7tAY0dtGd22nC6OEoYi/a11sdeDNiTx/b6MPIagbRhRA/8VewuUMLvh1h2SVMTCtj2JOQQfofS/j3l+YeuC9e5vuDH++vLz683p7efWydRrjIeZQEGFbpgi9SEbGCm4Yllo/ntvmbjSsho+c5UI9XAmb7by6gGASoTHM4kWEZpiELiSO1zdYo3RYTuAhrJiaBK0FNKEzQhO8XCbL5WK+mCyjFhpPF5P5hE4wmVF6cjI7mXbQlCzpnMxwktBkTqZJfR7WGYnfd9lrTHBwX1ue5Qfcgt8TeAXrsO7K97FjL7lPmK5sEITo9OxoDAt94F/Ew84G1MlH0CSEhB9G0Wjtm+ysdeQnUew11HdlZABj03IYm64e47sqe+QWZ2XZxnQdD0XXuTggkZ8e+XlDObKyBOS+ktL1aDo6W8dw/1+iUnvPLw/cSPbkru3o2ef6EGO8jmF7XNJqDVPL/IP5bw), I haven't yet seen a difference aside from the different aspect ratio (since the unit test renders to a `1x1` canvas.

The [particular unit test](http://localhost:8080/Specs/SpecRunner.html?spec=Scene%2FTimeDynamicPointCloud%20renders%20in%203D) does the following:
* renders the model, expects a red pixel (passes)
* adjusts the timeline and renders again, expects a red pixel (passes) 
* Moves the camera 10m to the right, expects nothing to render (fails)

I do know that it's something from `PointCloud.js` that causes the problem, because if I do 

```
git checkout main -- Source/Scene/PointCloud.js
```
then the test runs fine. 

Also I've tried increasing the moveRight amount, if I increase it to `30.0` the test passes.

I guess I'll read through `TimeDynamicPointCloud` closer as well as look at this diff in more depth since I'm not yet seeing the problem.

@lilleyse does anything jump out at you at what I might have done wrong in this file?

Reply:
Update: I figured out the problem. I wasn't passing in the typed array correctly when computing the approximate bounding volume. Despite most of the properties needed being undefined, somehow it computed _a_ bounding volume with an infinite radius.

I'm also not sure why this changes the unit test. My guess is that the camera zoom was incorrect due to the bad bounding sphere... but it was zooming to a specific Cartesian3 + heading-pitch-roll. Not sure if that makes use of bounding spheres.